devel/addon/kyverno/install.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The cert-manager Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Installs an instance of Kyverno with all Pod security policies enabled.
+# * https://kyverno.io/policies/pod-security/
+#
+# This ensures that the cert-manager deployment adheres to best practice Pod security.
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+SEI_VERSION="v0.1.0"
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")
+source "${SCRIPT_ROOT}/../../lib/lib.sh"
+
+check_tool kubectl
+check_tool helm
+check_tool kustomize
+
+# Install latest version of Kyverno
+helm repo add kyverno https://kyverno.github.io/kyverno/
+helm repo update
+helm upgrade --install --wait kyverno kyverno/kyverno --namespace kyverno --create-namespace
+# Install all Pod security policies
+# See https://kyverno.io/policies/pod-security
+kustomize build https://github.com/kyverno/policies/pod-security | kubectl apply -f -
```

Comment:
This policy causes the current cert-manager deployment to fail with a `Running as root is not allowed` error, as follows:


```
Release "cert-manager" does not exist. Installing it now.
Error: admission webhook "validate.kyverno.svc" denied the request: 
resource Deployment/cert-manager/cert-manager-cainjector was blocked due to the following policies
require-run-as-non-root:
  autogen-check-containers: 'validation error: Running as root is not allowed. The
    fields spec.securityContext.runAsNonRoot, spec.containers[*].securityContext.runAsNonRoot,
    and spec.initContainers[*].securityContext.runAsNonRoot must be `true`. Rule autogen-check-containers[0]
    failed at path /spec/template/spec/securityContext/runAsNonRoot/. Rule autogen-check-containers[1]
    failed at path /spec/template/spec/containers/0/securityContext/.'

```
-- https://prow.build-infra.jetstack.net/view/gs/jetstack-logs/pr-logs/pull/jetstack_cert-manager/4036/pull-cert-manager-e2e-v1-21/1395302762056519683#1:build-log.txt%3A335